crates/subspace-node/src/system_domain/chain_spec.rs

```diff
@@ -192,24 +230,52 @@ pub fn gemini_3d_config() -> ExecutionChainSpec<GenesisConfig> {
+                vec![
+                    (
+                        AccountId::from_ss58check(
+                            "5Df6w8CgYY8kTRwCu8bjBsFu46fy4nFa61xk6dUbL6G4fFjQ",
+                        )
                         .expect("Wrong executor account address"),
-                    1_000 * SSC,
-                    DomainConfig {
-                        wasm_runtime_hash: blake2b_256_hash(
-                            system_domain_runtime::CORE_PAYMENTS_WASM_BUNDLE,
+                        1_000 * SSC,
+                        DomainConfig {
+                            wasm_runtime_hash: blake2b_256_hash(
+                                system_domain_runtime::CORE_PAYMENTS_WASM_BUNDLE,
+                            )
+                            .into(),
+                            max_bundle_size: 4 * 1024 * 1024,
+                            bundle_slot_probability: (1, 1),
+                            max_bundle_weight: Weight::MAX,
+                            min_operator_stake: 100 * SSC,
+                        },
+                        AccountId::from_ss58check(
+                            "5Df6w8CgYY8kTRwCu8bjBsFu46fy4nFa61xk6dUbL6G4fFjQ",
                         )
-                        .into(),
-                        max_bundle_size: 4 * 1024 * 1024,
-                        bundle_slot_probability: (1, 1),
-                        max_bundle_weight: Weight::MAX,
-                        min_operator_stake: 100 * SSC,
-                    },
-                    AccountId::from_ss58check("5Df6w8CgYY8kTRwCu8bjBsFu46fy4nFa61xk6dUbL6G4fFjQ")
                         .expect("Wrong executor account address"),
-                    Percent::from_percent(10),
-                )],
+                        Percent::from_percent(10),
+                    ),
+                    (
+                        AccountId::from_ss58check(
+                            "5Df6w8CgYY8kTRwCu8bjBsFu46fy4nFa61xk6dUbL6G4fFjQ",
+                        )
+                        .expect("Wrong executor account address"),
+                        1_000 * SSC,
+                        DomainConfig {
+                            wasm_runtime_hash: blake2b_256_hash(
+                                system_domain_runtime::CORE_ETH_RELAY_WASM_BUNDLE,
+                            )
+                            .into(),
+                            max_bundle_size: 4 * 1024 * 1024,
+                            bundle_slot_probability: (1, 1),
+                            max_bundle_weight: Weight::MAX,
+                            min_operator_stake: 100 * SSC,
+                        },
+                        AccountId::from_ss58check(
+                            "5Df6w8CgYY8kTRwCu8bjBsFu46fy4nFa61xk6dUbL6G4fFjQ",
+                        )
+                        .expect("Wrong executor account address"),
+                        Percent::from_percent(10),
+                    ),
```

Comment:
I remember Jeremiah mentioned there is one genesis domain core-payments, so probably at least remove it from gemini-3d, so that we can also demostrate the feature of registering a new domain in a more formal environment.

Reply:
Okay. Removed it from Gemini-3d